datamade/committee-oversight
# Landing page

## Overview

Closes #82. This PR creates a new landing page and visual style for the site. It should look like this:

<img width="1388" alt="Screen Shot 2019-09-18 at 4 42 21 PM" src="https://user-images.githubusercontent.com/1094243/65188139-5376e980-da33-11e9-8d52-f5d4cd17fd95.png">

When a committee is selected from the search box, the site should redirect to that committee page.

This PR also includes an updated `hearings.dump` file.

## Testing Instructions

 * Stop your application containers and remove your Postgis volume: `docker-compose down && docker volume rm committee-oversight_committeeoversight-db-data`
* Restart your application: `docker-compose up`
* In another tab, load in data from `hearings.dump` (this will take a minute): `docker-compose exec postgres pg_restore -C -j4 --no-owner -U postgres -d hearings /app/hearings.dump`
* Navigate to http://localhost:8000/accounts/login/, login, play around, and make sure it looks like the above
* Use the search bar to find the House Committee on Science, Space, and Technology. Go to the page and make sure it shows you a Sample Committee Page
